### PR TITLE
showing place holder

### PR DESCRIPTION
**Pull request by ogallon**
Showing a place holder on the EditTextField

image

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -5,10 +5,10 @@
...
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (otigreros) [inline .idea/misc.xml]: line

File: .idea/misc.xml [hunk 2]
```diff
@@ -5,10 +5,10 @@
       <configuration PROFILE_NAME="Debug" CONFIG_NAME="Debug" />
     </configurations>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (otigreros) [inline .idea/misc.xml]: JDK_1_8

File: app/src/main/java/co/condorlabs/customcomponents/Constants.kt [hunk 3]
```diff
@@ -193,7 +193,7 @@ const val TITLE_TYPE = 1
 const val SUBTITLE_TYPE= 2
 const val SECTION_TITLE_TYPE = 3
 const val LINK_TYPE = 4
-
+const val MILLISECONDS_TO_SHOW_PLACE_HOLDER = 200.toLong()
```
Comment (EAlexisDuque) [inline app/src/main/java/co/condorlabs/customcomponents/Constants.kt]: **Suggestion**: `200L`

File: app/src/main/java/co/condorlabs/customcomponents/customedittext/BaseEditTextFormField.kt [hunk 4]
```diff
@@ -208,13 +215,30 @@ open class BaseEditTextFormField(context: Context, private val attrs: AttributeS
...
+        }
+    }
 
-            if (isValid.error.isNotEmpty()) {
-                showError(isValid.error)
-            } else {
-                clearError()
-            }
+    private fun showPlaceholder() {
+        editText?.postDelayed({
```
Comment (EAlexisDuque) [inline app/src/main/java/co/condorlabs/customcomponents/customedittext/BaseEditTextFormField.kt]: Is this **editText?** the same as **textInputLayout?.editText?** ??

File: app/src/main/res/values/attrs.xml [hunk 5]
```diff
@@ -34,6 +34,7 @@
         <attr name="min_lines" format="integer" />
         <attr name="background_alpha" format="integer" />
         <attr name="multiline" format="boolean" />
+        <attr name="placeholder" format="string" />
```
Comment (EAlexisDuque) [inline app/src/main/res/values/attrs.xml]: **Suggestion**: You should override `<attr name="android:hint" />`

File: app/src/main/java/co/condorlabs/customcomponents/customedittext/BaseEditTextFormField.kt (hunk 4 above)
Comment (ogallon) [reply app/src/main/java/co/condorlabs/customcomponents/customedittext/BaseEditTextFormField.kt]: sii

File: app/src/main/res/values/attrs.xml (hunk 5 above)
Comment (ogallon) [reply app/src/main/res/values/attrs.xml]: why do I want to do this ?

File: app/src/main/java/co/condorlabs/customcomponents/Constants.kt (hunk 3 above)
Comment (ogallon) [reply app/src/main/java/co/condorlabs/customcomponents/Constants.kt]: is the same whats the difference or the benefit to use it in this way ?

**Pull request merged**